logger/consoleFormatter.go

```diff
@@ -24,14 +24,30 @@ func (cf *ConsoleFormatter) Output(line LogLineHandler) []byte {
 
 	level := LogLevel(line.GetLogLevel())
 	levelColor := getLevelColor(level)
+	timestamp := displayTime(line.GetTimestamp())
+	loggerName := formatLoggerName(line.GetLoggerName())
```

Comment:
I think we should be able to turn on/off this string, just like correlation.

Reply:
Done.